ui/src/authorizations/apis/index.ts

```diff
@@ -0,0 +1,16 @@
+import AJAX from 'src/utils/ajax'
+
+export const createAuthorization = async authorization => {
+  try {
+    const {data} = await AJAX({
+      method: 'POST',
```

Comment:
should this be in the client library?

Reply:
yup.  just wanted to get this out quickly